test/Riok.Mapperly.Tests/Mapping/UnsafeAccessorTest.cs

```diff
@@ -143,6 +143,48 @@ public void ManualUnflattenedPropertyNullablePath()
             );
     }
 
+    [Fact]
+    public void PropertyWithPrivateSetter()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypes(
+            """
+                partial B Map(A source);
```

Comment:
This has weird indention… align the entire raw string at the same indention. Same for the other test.